ecs/alb.tf

```diff
@@ -48,3 +48,25 @@ resource "aws_alb_listener_certificate" "https" {
   listener_arn    = aws_alb_listener.https.arn
   certificate_arn = each.value.arn
 }
+
+resource "aws_lb_listener_rule" "main" {
+  count = length(var.alb_listener_rule)
```

Comment:
please make this `for_each` and pluralize `alb_listener_rules` to allow multiple